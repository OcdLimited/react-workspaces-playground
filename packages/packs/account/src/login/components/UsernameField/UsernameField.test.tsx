import React from 'react';
import { render } from '@testing-library/react';
import { UsernameField } from '.';
import { Formik } from 'formik';

it('renders without crashing', () => {
	render(
		<React.Fragment>
			<Formik onSubmit={() => {}} initialValues={{}}>
				{() => <UsernameField />}
			</Formik>
		</React.Fragment>,
	);
});

it('snapshot', () => {
	const { container } = render(
		<Formik onSubmit={() => {}} initialValues={{}}>
			{() => <UsernameField />}
		</Formik>,
	);
	expect(container).toMatchSnapshot();
});
