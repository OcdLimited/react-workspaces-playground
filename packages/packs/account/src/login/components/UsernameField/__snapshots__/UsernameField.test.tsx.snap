// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div
    class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
  >
    <label
      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink="false"
    >
      UserNameOrEmailAddress
    </label>
    <div
      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
    >
      <input
        aria-invalid="false"
        autocomplete="username"
        class="MuiInputBase-input MuiOutlinedInput-input"
        name="username"
        type="text"
        value=""
      />
      <fieldset
        aria-hidden="true"
        class="PrivateNotchedOutline-root-121 MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="PrivateNotchedOutline-legendLabelled-123"
        >
          <span>
            UserNameOrEmailAddress
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
